### ISSUE TITLE
Jensen wake model

### ISSUE THREAD

**Issue by gogannes** [NREL/floris#222]
**Bug description**

Dear all,
thanks again for your efforts in developing an outstanding FLORIS framework!

The current implementation of the Jensen wake deficit generates a quadratic wake deficit (see figure A). If I am not mistaken, the standard Jensen wake model should be circular (see figure B).

Figure A
![A](https://user-images.githubusercontent.com/23582486/123804802-d0641200-d8ed-11eb-95ef-96058c5ccc9b.png)

Figure B
![B](https://user-images.githubusercontent.com/23582486/123805974-cc84bf80-d8ee-11eb-87c9-7d5c6e7dba94.png)

Is there any specific reason for the current implementation? 

I believe not many are using the Jensen model, but it might still be used as reference...
I also think the difference between circular and quadratic in terms of power production of a downstream WT can be small in many situations. However, especially if the downstream turbine hub height is different to the wake sheding turbine, larger differences should occur.

**To Reproduce**
Code to reproduce figure A:
```
import matplotlib.pyplot as plt
import floris.tools as wfct
fi = wfct.floris_interface.FlorisInterface("examples/other_jsons/jensen.json")
fi.calculate_wake()
hor_plane = fi.get_hor_plane()
fig, axes = plt.subplots(2, 1)
wfct.visualization.visualize_cut_plane(hor_plane, ax=axes[0])
x_loc = 200
cross_plane = fi.get_cross_plane(x_loc=x_loc, z_bounds=[1, 200])
wfct.visualization.visualize_cut_plane(cross_plane, ax=axes[1])
axes[1].set_title("@ x=%.0f" % x_loc)
fig.tight_layout()
plt.show()
```
Code to reproduce figure B:
Add the following lines to jensen.py at line 122:
```
y_center = np.zeros_like(boundary_line) + turbine_coord.x2 + deflection_field
z_center = np.zeros_like(boundary_line) + turbine.hub_height
c[((y_locations - y_center) ** 2 + (z_locations - z_center) ** 2) > (boundary_line ** 2)] = 0
```

**Expected behavior**
See figure B.

**Floris Version**
2.3

**Comment by rafmudaf** [comment]
Hi @gogannes,

Good catch! I think your proposed changed would indeed fix the problem. Could you open a pull request to the `develop` branch with this change? Documentation for contributing to FLORIS via GitHub is available at https://floris.readthedocs.io/en/latest/source/developers.html.

Thanks again!

Rafael

**Comment by bayc** [comment]
Fixed with PR #226. Thanks @gogannes!
